nixos/modules/system/boot/resolved.nix

```diff
@@ -119,6 +119,48 @@ in
       '';
     };
 
+    services.resolved.dnsovertls = mkOption {
```

Comment:
The commit log should be `nixos/resolved: ...`

Reply:
I did